paddle/fluid/recordio/CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
 # internal library.
 cc_library(header SRCS header.cc)
-cc_test(header_test SRCS header_test.cc DEPS header)
+#cc_test(header_test SRCS header_test.cc DEPS header)
```

Comment:
It's better to add some `TODO` comments here, or maybe we will forget it.

Reply:
done